IDR/production/utils/get_ids.py

```diff
@@ -0,0 +1,127 @@
+import pathlib
+import requests
+import pandas as pd
+
+
+def get_id(json):
```

Comment:
I think I've seen a couple functions named similarly to this (e.g. `get_id`) - maybe this would be better defined as `get_id_from_json`? I am not sure if this comment is helpful

Reply:
oh, maybe I'm thinking of something I reviewed in the notebooks folder? 🤔 It's probably worth double checking

Reply:
Yes this was a function directly pulled from the notebook. Although, I do agree I use `get_whatever` a lot so I changed the name to your suggestion.

Added in ca4301b8fbdd621d227115a0e7f744b42bb53184.